src/connectors/range-slider/__tests__/connectRangeSlider-test.js

```diff
@@ -184,4 +184,40 @@ describe('connectRangeSlider', () => {
       expect(helper.search.callCount).toBe(2);
     }
   });
+
+  it('should add numeric refinement when refining min boundary', () => {
+    const rendering = sinon.stub();
+    const makeWidget = connectRangeSlider(rendering);
+
+    const attributeName = 'price';
+    const widget = makeWidget({ attributeName, min: 0, max: 500 });
+
+    const helper = jsHelper(fakeClient, '', widget.getConfiguration());
+    helper.search = sinon.stub();
+
+    widget.init({
+      helper,
+      state: helper.state,
+      createURL: () => '#',
+      onHistoryChange: () => {},
+    });
+
+    {
+      // first rendering
+      expect(helper.getNumericRefinement('price', '>=')).toEqual([0]);
+      expect(helper.getNumericRefinement('price', '<=')).toEqual([500]);
+
+      const renderOptions = rendering.lastCall.args[0];
+      const { refine } = renderOptions;
+      refine([10, 30]);
+
+      expect(helper.getNumericRefinement('price', '>=')).toEqual([10]);
+      expect(helper.getNumericRefinement('price', '<=')).toEqual([30]);
+      expect(helper.search.callCount).toBe(1);
+
+      refine([0, undefined]);
```

Comment:
Just for the sake of being exhaustive, maybe we could try without the bounds. And also do the third case with `[undefined, 20]`